cjs/shared/mime.js

```diff
@@ -7,26 +7,31 @@ const Mime = {
   'text/html': {
     docType: '<!DOCTYPE html>',
     ignoreCase: true,
+    isXml: false,
     voidElements: /^(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr)$/i
   },
   'image/svg+xml': {
     docType: '<?xml version="1.0" encoding="utf-8"?>',
     ignoreCase: false,
```

Comment:
I think I've named it as such due XHTML but again I am not sure why we need to change that ... it could be breaking if anyone out there brand-check that property.

Reply:
> why don't we escape all html entities

I think you are better off with JSDOM there ... it's 100% standard, and 100% slower than LinkeDOM

Reply:
to be fair, XML is no tthe main target here and you are suggesting XML related changes and paths that would slow the common use case by far, as example suggesting to escape all html entities ... I understand this is valuable for your business but this project is about perf ... perf for most sommon use cases, which is not XML, so apologies if my answers are not the most welcoming one, but I am trying to preserve the original ideal of this project which is: work for most common cases and as fast as possible ... your quest feels a bit against that initial/original goal, hence my nitpicking in here. Hope you can understand, and hope your fork will work great for your use case too!